doc/sphinx/using/libraries/funind.rst

```diff
@@ -242,39 +249,28 @@ Tactics
       This error may be raised when some inversion lemma failed to be generated by
       Function.
 
-
-   .. tacv:: functional inversion @natural
-
-      This does the same thing as :n:`intros until @natural` followed by
-      :n:`functional inversion @ident` where :token:`ident` is the
-      identifier for the last introduced hypothesis.
-
-   .. tacv:: functional inversion @ident @qualid
-             functional inversion @natural @qualid
-
-      If the hypothesis :token:`ident` (or :token:`natural`) has a type of the form
-      :n:`@qualid__1 {+ @term__i } = @qualid__2 {+ @term__j }` where
-      :n:`@qualid__1` and :n:`@qualid__2` are valid candidates to
-      functional inversion, this variant allows choosing which :token:`qualid`
-      is inverted.
-
 .. _functional-scheme:
 
 Generation of induction principles with ``Functional`` ``Scheme``
 -----------------------------------------------------------------
 
 
-.. cmd:: Functional Scheme @ident__0 := Induction for @ident' Sort @sort {* with @ident__i := Induction for @ident__i' Sort @sort}
+.. cmd:: Functional Scheme @func_scheme_defn {* with @func_scheme_defn }
+
+   .. insertprodn func_scheme_defn func_scheme_defn
+
+   .. prodn::
+      func_scheme_defn ::= @ident := Induction for @qualid Sort @sort_family
```

Comment:
What is the "n" in "defn" for? If this is just for "definition", then maybe we should favor a consistent abbreviation everywhere, and that would rather be "def".

Reply:
I think "defn" is the more common casual abbreviation for "definition" though I can't find any reference that discusses "def" or "defn".  Somewhat similar is "govt" as an abbreviation for "government".  I'll make this one "def"; there are a few other used of "def" in nonterminal names.